tests/test_apython.py

```diff
@@ -168,3 +181,17 @@ def test_apython_non_existing_module(capfd):
     out, err = capfd.readouterr()
     assert out == ''
     assert "No module named idontexist" in err
+
+def test_apython_pythonstartup(capfd, use_readline, monkeypatch, tempfd):
+
+    monkeypatch.setenv('PYTHONSTARTUP', tempfd.name)
+    tempfd.write(startupfile.encode())
+    tempfd.flush()
+
+    with patch('sys.stdin', new=io.StringIO(
+            'print(foo)\nprint(hehe())\n')):
+        with pytest.raises(SystemExit):
+            apython.run_apython(['--banner=test'] + use_readline)
+    out, err = capfd.readouterr()
+    assert out == ''
+    assert err == 'test\n>>> 1\n>>> 42\n>>> \n'
```

Comment:
Great, thanks for writing the test! :)

Reply:
For what its worth, I broke the test when i made it more complex when testing the windows readline behavior? Not certain how best to address that, since stdout doesn't seem to be populated with some output from calling a function.

Reply:
Nevermind - i fixed the test vectors used.